src/invidious/comments.cr

```diff
@@ -481,11 +481,15 @@ def replace_links(html)
     url = URI.parse(anchor["href"])
 
     if {"www.youtube.com", "m.youtube.com", "youtu.be"}.includes?(url.host)
-      if url.path == "/redirect"
-        params = HTTP::Params.parse(url.query.not_nil!)
-        anchor["href"] = params["q"]?
+      if url.host == "youtu.be"
+        url = "/watch?v=#{url.request_target.lstrip('/')}"
```

Comment:
You probably don't want to use `request_target` here, as it returns the path and query concatenated.
E.g, for the URL: `youtube.be/abcdef12345?t=20s`, that will give `/abcdef12345?t=20s`, meaning that you'll get double query parts (`?watch=abcdef12345` and `?t=20s`)